app/src/main/java/com/launcher/silverfish/common/LG.java

```diff
@@ -0,0 +1,63 @@
+package com.launcher.silverfish.common;
+
+import android.util.Log;
+
+/** Intelligent logger optionally provides calling class & method for the message. <br>
+ Standard usage: lg("any string");  [Will generate a TAG of Class.methodName ] <br>
+ @author Matt Arnold */
+public class LG {
+
+    private static boolean debug = false;
+    public static void setDebug(boolean debug) {LG.debug = debug;}
```

Comment:
I'm sure the code reformat would like spaces between `{}`. Just being picky here though.